__tests__/unit/old-structure/tree-fetching/stratetegies/beagle-with-fallback-to-cache.spec.ts

```diff
@@ -135,35 +133,10 @@ describe('Utils: tree fetching (load: beagle-with-fallback-to-cache)', () => {
     expect(nock.isDone()).toBe(true)
   })
 
-  it(
-    'should throw error if received 304 from bff and doesn\'t find item in the storage',
-    async () => {
-      const metadata = {
-        beagleHash: 'testing',
-        requestTime: 20203030,
-        ttl: '5'
-      }
-      
-      const headers = { 'beagle-hash': 'testing', 'cache-control': 'max-age=5' }
-      nock(basePath).get(path).reply(304, undefined, headers)
-      const onChangeTree = jest.fn()
-      await expect(viewClient.load({ url, onChangeTree, retry })).rejects.toEqual([
-        new BeagleExpiredCacheError(url),
-        new BeagleCacheError(url),
-        new BeagleCacheError(url)
-      ])
-      expect(storage.getItem).toHaveBeenCalledWith(cacheKey)
-      expect(storage.getItem).toHaveBeenCalledWith(treeKey)
-      expect(nock.isDone()).toBe(true)
-    },
-  )
-
   it('should throw errors', async () => {
     nock(basePath).get(path).reply(500, JSON.stringify({ error: 'unexpected error' }))
     await expect(viewClient.load({ url, onChangeTree: jest.fn(), retry })).rejects.toEqual([
-      new BeagleExpiredCacheError(url),
-      new BeagleNetworkError(url, {} as Response),
-      new BeagleCacheError(url),
+      new BeagleNetworkError(url, {} as Response, 500,'GET'),
```

Comment:
```suggestion
      new BeagleNetworkError(url, {} as Response, 500, 'GET'),
```
It would be nice to add the space after the "," to keep a pattern.